### PR TITLE
chore(binding-opcua): enable strict type checking

### PR DESCRIPTION
**Pull request by danielpeintner**
another one of https://github.com/eclipse-thingweb/node-wot/issues/758

Note: since at the moment I am not able to test OPCUA locally due to network/company restriction I will use the CI for testing.

### CODE REVIEW

File: packages/binding-opcua/src/codec.ts [hunk 1]
```diff
@@ -268,7 +266,9 @@ export class OpcuaBinaryCodec implements ContentCodec {
         const binaryStream = new BinaryStream(bytes);
         const dataValue = new DataValue();
         dataValue.decode(binaryStream);
-        return jsonify(opcuaJsonEncodeDataValue(dataValue, true));
+        // TODO wrapping JSON.parse(JSON.stringify(a)) a la jsonify needed?
```
[the remark this answers is not in the document]
Comment (relu91) [reply packages/binding-opcua/src/codec.ts:269]: Mmm, I don't see any immediate reason that would require us to clone the object. I am ok with this change.

[the remark this answers is not in the document]
Comment (danielpeintner) [reply packages/binding-opcua/src/codec.ts:269]: let's see whether @erossignon has an opinion

**Comment by danielpeintner** [comment]
@erossignon seems to be busy.  Anyhow, the changes seem simple enough to go on... what do others think?

**Comment by relu91** [comment]
Thinking on the point, the only reason that I see to clone `dataValue` might be to sanitize the Object and remove functions and methods. Can we still keep this conversion and mark it as an issue? 🤔

**Comment by danielpeintner** [comment]
> Just to keep the current idempotent with the previous implementation.

Yes, let's keep it, see https://github.com/eclipse-thingweb/node-wot/pull/1057/commits/732baa636e3ad41e12ae3e5a289ba430aa4da811
I don't think an issue is really needed.

**Comment by erossignon** [comment]
Sorry, I didn't find time to look into this; but you did well any, we can always look at why this JSON.parse(JSON.stringify(opcuaJsonEncodeDataValue(dataValue, true))); is needed.
I think we can simplify this with the most recent version node-opcua-json package that improved the opcuaJsonEncodeDataValue. but that's a different story. 
I'll propose a PR.

**Pull request merged**